scripts/multinode/configure-sec.sh

```diff
@@ -50,8 +52,14 @@ function configure_microshift() {
         sudo systemctl stop --now kubelet
     fi
     sudo systemctl reset-failed kubelet || true
-    sudo pkill -9 --exact kubelet       || true
-    sudo rm -rf /var/lib/kubelet &> /dev/null || true
+    until ! pidof kubelet &>/dev/null ; do
+        sudo pkill -9 --exact kubelet || true
+        sleep 1
+    done
```

Comment:
I am wondering if there is a need for a loop, given that sigkill is not catchable/ignorable.

Reply:
The idea was to make sure that the process is dead. Let me reqwrite the code not to send KILL more than once.